mcrouter/scripts/docker/container/ubuntu-packaged/usr/local/bin/entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -x
+
+
+function write_default_config() {
+cat << EOF > /etc/mcrouter.json
+{
+  "pools": {
+    "A": {
+      "servers": [
+        "127.0.0.1:5001"
+      ]
+    }
+  },
+  "route": "PoolRoute|A"
+}
+EOF
+}
+
+function run_mcrouter() {
+        local PORT=$1
+        mcrouter \
+        --config file:/etc/mcrouter.json
+#        -p "${PORT}"
```

Comment:
uncomment -p and add a "$@" to the end so other command line options can be passed to mcrouter